### DISCUSSION TITLE
Add ability to calculate properties for above and below the neutral/shear axis

### DISCUSSION THREAD

**Discussion by EngDM** [robbievanleeuwen/section-properties#452] [Ideas]
There are some pretty niche calculations required for monosymmetric beams that involve splitting the member at it's shear center and re-calculating centroids for the shape above and below this axis. It would be nice if there was a way to feed an x-y coordinate to the analysis call and it calculates properties based on this input, returning values for above-below, left-right of this specified coordinate. If it accepts an input of coordinate instead of just assuming shear center, then we could pass it the shear center OR a coordinate.

This could also be achieved by creating new sections for top and bottom, although I'm not sure how you'd go about splitting the sections.

**Comment by robbievanleeuwen** [comment]
You can already [split](https://sectionproperties.readthedocs.io/en/stable/user_guide/geometry.html#split) a `Geometry` object either side of a line. This method returns `Geometry` objects either side of the line.
